packages/amplify-cli-core/src/errors/amplify-exception.ts

```diff
@@ -145,36 +146,38 @@ export type AmplifyErrorType =
   | 'ResourceAlreadyExistsError'
   | 'ResourceInUseError'
   | 'ResourceNotReadyError'
+  | 'SearchableMockProcessError'
+  | 'SearchableMockUnavailablePortError'
+  | 'SearchableMockUnsupportedPlatformError'
   | 'StackNotFoundError'
   | 'StackStateError'
-  | 'UserInputError'
-  | 'MockProcessError'
-  | 'SearchableMockUnsupportedPlatformError'
-  | 'SearchableMockUnavailablePortError'
-  | 'SearchableMockProcessError';
+  | 'UserInputError';
 
 /**
  * Amplify fault types
  */
 export type AmplifyFaultType =
-  | 'AnalyticsCategoryFault'
   | 'AmplifyBackupFault'
+  | 'AnalyticsCategoryFault'
+  | 'AuthCategoryFault'
+  | 'BackendDeleteFault'
   | 'BackendPullFault'
   | 'ConfigurationFault'
-  | 'BackendDeleteFault'
   | 'ConfigurationFault'
   | 'DeploymentFault'
+  | 'DeploymentStateUploadFault'
```

Comment:
This is the only functional addition in this file, the rest of the changes are due to sorting.